tests/performance_tests/README.md

```diff
@@ -0,0 +1,605 @@
+# Performance Harness Tests
+
+The `performance_test.py` script performs a binary search of EOS Token Transfers Per Second (TPS) range at configurable low test duration scoring each individual test scenario to find where TPS seems to be topping out. It does this by iteratively configuring and running `performance_test_basic.py` tests and analyzing the output to determine a success metric to continue the search.  The script then proceeds to conduct an additional search with longer duration test runs at a narrowed TPS window around the discovered maximum TPS throughput. Finally it produces a report on the entire performance run, summarizing each individual test scenario, results, and full report details on the tests when maximum TPS was achieved ([Performance Test Report](#performance-test))
+
+The `performance_test_basic.py` script performs a single basic performance test that targets a configurable TPS target and, if successful, reports statistics on perfomance metrics measured during the test.  It configures and launches a blockchain test environment, creates wallets and accounts for testing, configures and launches transaction generators for creating specific transaction load in the ecosystem.  Finally it analyzes the performance of the system under the configuration through log analysis and chain queries and produces a [Performance Test Basic Report](#performance-test-basic).
+
+The `launch_generators.py` script provides a means to easily calculate and spawn the number of transactions generator instances to generate a given target TPS, distributing generation load between the instances in a fair manner such that the aggregate load meets the requested test load.
```

Comment:
"number of transactions generator" -> "number of transaction generator"

Reply:
Somehow I missed these in my first pass.  Addressed: [f7aff36](https://github.com/AntelopeIO/leap/pull/352/commits/f7aff36f3465a311ac2393ac1611cd54e931225a)